tests/unit/ui/components/TestRating.js

```diff
@@ -142,13 +142,50 @@ describe(__filename, () => {
     verifyRating(root);
   });
 
+  it('renders an appropriate title when updating a rating', () => {
+    const root = render({ rating: 3 });
+
+    [1, 2, 3].forEach((rating) => {
+      expect(root.ratingElements[rating].className).toEqual('Rating-choice Rating-selected-star');
+      expect(root.ratingElements[rating].title).toEqual(`Update your rating to ${rating} out of 5.`);
+    });
+    [4, 5].forEach((rating) => {
+      expect(root.ratingElements[rating].className).toEqual('Rating-choice');
+      expect(root.ratingElements[rating].title).toEqual(`Update your rating to ${rating} out of 5.`);
+    });
+  });
+
   it('rounds ratings to nearest 0.5 multiple', () => {
     // This should be treated like a rating of 3.5 in text.
-    const root = render({ rating: 3.60001 });
+    const root = render({ rating: 3.60001, readOnly: true });
 
     expect(findDOMNode(root).title).toContain('3.6 out of 5');
   });
 
+  it('converts rating numbers to a float', () => {
```

Comment:
We don't have any code that explicitly converts to float, so we don't really need this test. I believe a similar one was removed in an earlier commit.